client/bin/entrypoint.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -xeuo pipefail
 
+sleep 120
```

Comment:
I'd really rather not have slow things down purposely. What's exactly not ready? I'd prefer to add a check loop instead.

Reply:
IIRC in this case it has been cascade of DB and Koji hub. I guess that factor is that I'm using VM that is "uderpowered" by openshift requirements(4laptopcores/8GB/ssd storage). 
Possibly polling loop ? Still think that this is minimal slow down compared to the image rebuild, etc and greatly improves reliability.

Reply:
Let's try a polling loop, if it proves to be challenging to implement we can use this workaround.

I'm pushing back on this because I haven't really seen the need for this in my testing, but I only test with rhel 7 which may be the reason.

This project used to be scattered with `sleep` statements without any sort of comment or hint for their need. Such things make it more difficult to maintain the project and to ensure that problems are being addressed in an efficient way. Sometimes `sleep` is the most straightforward way of solving a problem but it really should be handled as a last resort.